src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/config/ManageUsersApiExceptionHandler.kt

```diff
@@ -60,6 +68,21 @@ class HmppsManageUsersApiExceptionHandler {
       )
   }
 
+  @ExceptionHandler(MethodArgumentNotValidException::class)
+  fun handleValidationException(e: MethodArgumentNotValidException): ResponseEntity<ErrorResponse> {
+    val field = if (e.allErrors.size > 0) e.allErrors[0].objectName else "none"
```

Comment:
Probably not important for this service, but in court-service I did a bit of work to ensure all errors from API where returned to client - I think I added an extra element in the ErrorResponse object.

Ignore for now, but might be important in the future

Reply:
Will take a look and see if I can do  something similar